src/pretalx/common/management/commands/init.py

```diff
@@ -90,8 +92,8 @@ def handle(self, *args, **options):
             ),
         )
         self.stdout.write(self.style.SUCCESS("\nNow that this is done, you can:"))
-        self.stdout.write(" - Create your first event at {event_url}").format(
-            event_url=event_url
+        self.stdout.write(
+            " - Create your first event at {event_url}".format(event_url=event_url)
```

Comment:
I think all of these could be f-strings now that they aren't translated anymore, I'll take care of that, too